docs/root/intro/version_history.rst

```diff
@@ -6,6 +6,7 @@ Version history
 * access log: added a :ref:`JSON logging mode <config_access_log_format_dictionaries>` to output access logs in JSON format.
 * access log: added dynamic metadata to access log messages streamed over gRPC.
 * admin: added support for displaying subject alternate names in :ref:`certs<operations_admin_interface_certs>` end point.
+* admin: /server_info now responds with a JSON object instead of a single string.
```

Comment:
nit: ref link to the CLI option. There is `:option:` syntax IIRC.

Reply:
What CLI option? You mean the admin endpoint?

Reply:
Sorry yes I meant the admin endpoint. Nevermind about `:option:`